packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -175,6 +258,45 @@ export default class ScriptTransformer {
     return undefined;
   }
 
+  private async _getTransformerAsync(filename: Config.Path) {
+    if (!this._config.transform || !this._config.transform.length) {
+      return null;
+    }
+
+    const transformPath = this._getTransformPath(filename);
+
+    if (!transformPath) {
+      return null;
+    }
+
+    const cached = this._transformCache.get(transformPath);
+    if (cached) {
+      return cached;
+    }
+
+    let transformer: AsyncTransformer = await import(transformPath);
+
+    if (!transformer) {
+      throw new TypeError('Jest: a transform must export something.');
+    }
+    const transformerConfig = this._transformConfigCache.get(transformPath);
+    if (typeof transformer.createTransformer === 'function') {
```

Comment:
I think we should expect the imported stuff to be on `.default` since it should be an object

Reply:
✅